tests/unit/test_run_logprep.py

```diff
@@ -34,8 +45,61 @@ def test_quickstart_rules_are_valid(self):
             "logprep",
             "--disable-logging",
             "--validate-rules",
+            "--config",
             "quickstart/exampledata/config/pipeline.yml",
         ]
         with pytest.raises(SystemExit) as e_info:
             run_logprep.main()
         assert e_info.value.code == 0
+
+    def test_version_arg_prints_logprep(self, capsys):
```

Comment:
this also tests for the case that no config version is specified, which should be reflected in the test name.